### PR TITLE
🏃 Refactor/cleanup load balancer related code

### PR DESCRIPTION
**Pull request by seanschneeweiss**
**What this PR does / why we need it**:
Overall this PR only refactors code to clean up some parts.

* This PR fixes the notation for load balancer related logs. The split notation `load balancer` is used.
* Also this PR restructures the `GetOrCreateFloatingIP` function for easier readability and removes a possible nil reference in the log line.
* This PR also renames some variables that shadowed imported package names.

**TODOs**:

- [x] squashed commits

/hold
<sub>Sean Schneeweiss <sean.schneeweiss@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

### CODE REVIEW

File: pkg/cloud/services/networking/floatingip.go [hunk 1]
```diff
@@ -30,28 +30,28 @@ import (
...
-		record.Eventf(openStackCluster, "SuccessfulCreateFloatingIP", "Created floating IP %s with id %s", fp.FloatingIP, fp.ID)
+		// only admin can add ip address
+		fpCreateOpts.FloatingIP = ip
+	}
 
+	fp, err = floatingips.Create(s.client, fpCreateOpts).Extract()
+	if err != nil {
+		record.Warnf(openStackCluster, "FailedCreateFloatingIP", "Failed to create floating IP %s: %v", ip, err)
+		return nil, err
 	}
```
Comment (sbueringer) [inline pkg/cloud/services/networking/floatingip.go:53]:
```suggestion
	}

```

Comment (seanschneeweiss) [reply pkg/cloud/services/networking/floatingip.go:53]: Done. Thank you for the very fast feedback :)

**Comment by sbueringer** [comment]
/lgtm
@seanschneeweiss just good timing as usual :)

File: pkg/cloud/services/networking/floatingip.go [hunk 2]
```diff
@@ -30,28 +30,29 @@ import (
...
+
+	fpCreateOpts.FloatingNetworkID = openStackCluster.Status.ExternalNetwork.ID
 
+	fp, err = floatingips.Create(s.client, fpCreateOpts).Extract()
+	if err != nil {
+		record.Warnf(openStackCluster, "FailedCreateFloatingIP", "Failed to create floating IP %s: %v", ip, err)
+		return nil, err
 	}
+
+	record.Eventf(openStackCluster, "SuccessfulCreateFloatingIP", "Created floating IP %s with id %s", ip, fp.ID)
```
Comment (hidekazuna) [inline pkg/cloud/services/networking/floatingip.go:55]: ip argument is optional, so ip must be `fp.FloatingIP` here.

**Review by hidekazuna — CHANGES_REQUESTED** [review]

File: pkg/cloud/services/networking/floatingip.go (hunk 2 above)
Comment (seanschneeweiss) [reply pkg/cloud/services/networking/floatingip.go]: Thank you, I've changed it accordingly.

**Pull request merged**